### DISCUSSION TITLE
Welcome to htmxpress Discussions!

### DISCUSSION THREAD

**Discussion by svandragt** [svandragt/htmxpress#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jasalt** [comment]
Hello, I'm Jarkko Saltiola, programming, music and camera geek and freelancer from Finland.

Got into web development during University (2011-) with Python Flask, AngularJS and Clojure(script). Did some small scripting jobs here and there and around 2016-2017 started getting more stable commercial gigs from a couple medium size local (web) stores that were migrating from Magento to WooCommerce. Have been mostly working with PHP and database side of WordPress since, but last year built a block editor sidebar plugin and became somewhat familiar with that side.

Recently started building some custom web apps with Django and HTMX and got immediately curious after finding HTMXPress project if the same tool could be used on both sides to save some mental overhead.


Thank's for the efforts on the project, it has been looking promising so far for what I have had time to play with it!

**Reply by svandragt** [reply]
Thanks and welcome!
